### PR TITLE
[LSX-2290] Add privacy manifest for user defaults

### PR DESCRIPTION
**Pull request by ilanahaddad**
### Issue Link :link: 

[Jira Ticket](https://lightspeedhq.atlassian.net/browse/LSX-2290)

Apple listed AFNetworking as one of the repositories who need to add a privacy manifest to their library.
Since the repo is deprecated, i'm adding it here so we can continue to use this library.

https://developer.apple.com/support/third-party-SDK-requirements/

### Goals :soccer:
Allows us to continue using this library, adhering to apple's requested changes needed to be made before May 1st 2024

### Implementation Details :construction:
Add privacy manifest that explicitly states the reasons for use of UserDefaults

### CODE REVIEW

**Review by ls-spencer-mohr — APPROVED** [review]

File: AFNetworking/PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jmortimer13) [inline AFNetworking/PrivacyInfo.xcprivacy]: Does this file need to be in a particular place to be correctly consumed/reviewed by Apple like under Resources or Support Files or Settings.bundle? Assuming this is ok, just want to set a precedent for consistency across internal repos.

Comment (ilanahaddad) [reply AFNetworking/PrivacyInfo.xcprivacy]: ya true i'll put it in a folder called Resources for consistency across other repos. not sure if it affects Apple's review but better safe than sorry!

Comment (ilanahaddad) [reply AFNetworking/PrivacyInfo.xcprivacy]: oh wait i had put it in Supporting Files not sure why that's not being reflected here ...

Comment (ilanahaddad) [reply AFNetworking/PrivacyInfo.xcprivacy]: oh right i think xcode automatically groups these in a folder called Supporting Files on xcode but its not a real folder in Finder so it's confusing. ill just move it to a new Resources folder

**Pull request merged**